packages/openactive-integration-tests/test/reporter.js

```diff
@@ -78,7 +86,7 @@ class Reporter {
         }
       }
     } catch (exception) {
-      console.log(testResult);
+      console.trace(testResult);
```

Comment:
I had no idea where this error was coming from. `console.trace` gives a stack trace which is helpful for logs that have very little context